### PR TITLE
Monthly update for Oct 2021

### PR DESCRIPTION
**Pull request by mbasmanova**
Add Monthly Updates section to the documentation and fill in the update
for Oct 2021.

### CODE REVIEW

File: velox/docs/monthly-updates/october-2021.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Presto Functions
+----------------
+
+* Add all remaining bitwise_xxx functions.
+* Add :func:`greatest` and :func:`least` functions.
+* Add :func:`log2` and :func:`log10` functions.
+* Add :func:`date_trunc`, :func:`day_of_week`, :func:`day_of_year`, :func:`year`, :func:`month`, :func:`day`, :func:`minute`, :func:`hour`, :func:`second` functions.
+* Add :func:`regexp_extract_all` function.
+* Add :func:`reverse`(varchar), :func:`split` functions.
```
Comment (majetideepak) [inline velox/docs/monthly-updates/october-2021.rst:35]: This line is being rendered incorrectly.

File: velox/docs/monthly-updates/october-2021.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+-------------
+
+* Add new article: :doc:`Plan Nodes and Operators <../develop/operators>`.
+
+Core Library
+------------
+
+* Enhance simple functions to
+
+  * allow returning an array of strings;
```
Comment (majetideepak) [inline velox/docs/monthly-updates/october-2021.rst:15]: full stop instead of semi-colon here and below 2 items?

**Review by majetideepak — COMMENTED** [review]
+1. Thanks for initiating this.

File: velox/docs/monthly-updates/october-2021.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  * process constant inputs once;
+  * access query configs;
+  * allow for providing fast path for all-ASCII inputs.
+
+  More details in :doc:`How to add a scalar function? <../develop/scalar-functions>` guide.
+* Add support for RIGHT OUTER joins.
+* Add DATE type.
+* Add FixedSizeArray vector.
+* Add support for reading DWRF files from `S3 <https://aws.amazon.com/pm/serv-s3/>`_.
```
[the remark this answers is not in the document]
Comment (mbasmanova) [reply velox/docs/monthly-updates/october-2021.rst:24]: > We can say AWS S3 API compatible file systems eg: Minio, AWS-S3?

Let me update.

> And technically Parquet files should work as well. But yea, we did not test explicitly.

Let's add automated tests for this first.

**Comment by mbasmanova** [comment]
@majetideepak Deepak, thank you for review. Addressed comments and updated the PR.

**Pull request closed without merging**